examples/acceptFromIDs.js

```diff
@@ -0,0 +1,63 @@
+// Required Modules
+const SteamUser = require('steam-user'),
+const SteamTotp = require('steam-totp'),
+const TradeOfferManager = require('steam-tradeoffer-manager'),
```

Comment:
You're within the module directory - `require('../lib/index.js');` - don't forget to leave a comment to replace it if used in production.